internal/serverconfig/config.go

```diff
@@ -4,6 +4,11 @@ import (
 	"strconv"
 )
 
+// The metadata key that stores the runner id associated with a client. This is
+// used by the CLI to advertise it's local client for when the server needs to
+// spawn jobs back on that client in response to an RPC.
+const GrpcMetadataRunnerId = "waypoint-runner-id"
```

Comment:
I think this should go in `internal/server` since that's the package that actually uses this?

Reply:
So I didn't want to pull the whole server back into `internal/client` that's why I put it here. I can move it where ever.

Reply:
That is a really good point. How about a new `internal/server/grpcmetadata` package or something? It feels weird that the inject/extract and key are all in separate packages. :)